vit-servicing-station-tests/src/common/clients/rest.rs

```diff
@@ -144,11 +145,33 @@ impl RestClient {
         serde_json::from_str(&content).map_err(RestError::CannotDeserialize)
     }
 
+    pub fn proposals_by_voteplan_id_and_index(
+        &self,
+        request: ProposalsByVoteplanIdAndIndex,
+    ) -> Result<Vec<FullProposalInfo>, RestError> {
+        let request_as_string = serde_json::to_string(&request)?;
+        serde_json::from_str(&self.post(&self.path_builder().proposals(), request_as_string)?)
+            .map_err(RestError::CannotDeserialize)
+    }
+
     pub fn fund_raw(&self, id: &str) -> Result<Response, RestError> {
         self.get(&self.path_builder().fund(id))
             .map_err(RestError::RequestError)
     }
 
+    pub fn challenges_raw(&self) -> Result<Response, RestError> {
+        self.get(&self.path_builder().challenges())
+            .map_err(RestError::RequestError)
+    }
+
+    pub fn challenges(&self) -> Result<Vec<Challenge>, RestError> {
+        let response = self.challenges_raw()?;
+        self.verify_status_code(&response)?;
+        let content = response.text()?;
+        self.logger.log_text(&content);
+        serde_json::from_str(&content).map_err(RestError::CannotDeserialize)
+    }
+
```

Comment:
Is this unrelated?

Reply:
oh it is, but i realized that in order to create integration tests in vit-testing on this commit i need to use more methods in rest .